example/provider-local/shoot.yaml

```diff
@@ -26,7 +26,6 @@ spec:
       maxSurge: 1
       maxUnavailable: 0
   kubernetes:
-    version: 1.27.1
```

Comment:
Please adapt `example/provider-local/shoot-workerless.yaml` as well.

Reply:
Hm, the file was already adapted 🤔